CIP-0140/README.lagda.md

```diff
@@ -0,0 +1,1568 @@
+---
+CIP: 140
+Title: Ouroboros Peras - Faster Settlement
+Category: Consensus
+Status: Proposed
+Authors:
+  - Arnaud Bailly <arnaud.bailly@iohk.io>
+  - Brian W. Bush <brian.bush@iohk.io>
+  - Sandro Coretti-Drayton <sandro.coretti@iohk.io>
+  - Yves Hauser <yves.hauser@iohk.io>
+  - Hans Lahe <hans.lahe@iohk.io>
+Implementors: Intersect
```

Comment:
```suggestion
Implementors:
  - Intersect
Proposed Solutions:
  - CPS-0017
```
In both cases entries are bulleted lists.  (I think it's possible that someday another solution might come along for this CPS.)

Reply:
p.s. I'm also now wondering if this proposal would also solve the candidate problem statement
- https://github.com/cardano-foundation/CIPs/pull/925

Please post if you think so and I'll suggest appropriate markup; if CIPs or CIPs aren't merged yet, the convention is to add a link to the PR like this (https://github.com/cardano-foundation/CIPs/pull/947/files) and then update after merge.

Reply:
Fixed in f3e8326e8f2812648fafc657a0d374c9033976bb and 4b3786049b38cfbc4b3bd8f0793a744f0d38a27d.